src/_pytest/nodes.py

```diff
@@ -61,11 +62,13 @@ def __get__(self, obj, owner):
         if obj is None:
             return self
 
-        # TODO: reenable in the features branch
-        # warnings.warn(
-        #     "usage of {owner!r}.{name} is deprecated, please use pytest.{name} instead".format(
-        #         name=self.name, owner=type(owner).__name__),
-        #     PendingDeprecationWarning, stacklevel=2)
+        warnings.warn(
```

Comment:
i had massive issue about one part of those warnings being triggered from pytest internals by accident which is why i deffered it on the old warnings system

Reply:
`catch_warnings` to the rescue. 😁 

It won't show the warnings for this specific access, but I guess that's OK because the whole class will be kicked out anyway. 👍